packages/core/src/rum/instrumentation/DdRumResourceTracking.tsx

```diff
@@ -234,6 +270,31 @@ export class DdRumResourceTracking {
         }
     }
 
+    static getTracingAttributes(url: string): DdRumResourceTracingAttributes {
+        try {
+            const hostname = new URL(url).hostname;
```

Comment:
URL.hostname is not yet implemented in RN: https://github.com/facebook/react-native/blob/main/Libraries/Blob/URL.js#L174